osmo_jupyter/db_access.py

```diff
@@ -0,0 +1,50 @@
+
+def connect_to_db():
+    # TODO
+
+def load_calculation_details(db_engine, node_ids, start_utc, end_utc, include_hub_id=False, downsample_interval=None):
+    """ Load node data from the calculation_details table, optionally with hub ID included from the readings table
+
+    Args:
+        db_engine: database engine created using `connect_to_db`

```

Comment:
Is there an argument for why `connect_to_db()` shouldn't just happen within this function? Do we only want to make the connection once and load data multiple different times within a notebook?

Reply:
yeah, plus I think I'm gonna prompt the user for the db password when we connect.